homeassistant/components/zwave_js/humidifier.py

```diff
@@ -150,10 +151,9 @@ def __init__(
     @property
     def is_on(self) -> bool | None:
         """Return True if entity is on."""
-        return int(self._current_mode.value) in [
-            self.entity_description.on_mode,
-            HumidityControlMode.AUTO,
-        ]
+        if (value := get_value_of_zwave_value(self._current_mode)) is None:
```

Comment:
It's the `value` attribute of `self._current_mode` that is optional. We need to guard for that. We don't need to use `get_value_of_zwave_value`.

Reply:
Updated